lib/rspec/core/invocations.rb

```diff
@@ -49,7 +49,24 @@ def bisect_formatter_for(argument)
       # @private
       class PrintVersion
         def call(_options, _err, out)
-          out.puts RSpec::Core::Version::STRING
+          overall_version = RSpec::Core::Version::STRING
+          unless overall_version =~ /[a-zA-Z]+/
+            overall_version = overall_version.split('.').first(2).join('.')
+          end
+
+          out.puts "RSpec #{overall_version}"
+
+          [:Core, :Expectations, :Mocks, :Rails, :Support].each do |const_name|
+            lib_name = const_name.to_s.downcase
+            begin
+              require "rspec/#{lib_name}/version"
+            rescue LoadError
+              out.puts "  - rspec-#{lib_name} not installed"
```

Comment:
I'm not sure we need to bother with this, if it's not installed just don't list it/